packages/Guard/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@paprika/guard",
+  "version": "0.1.0",
+  "description": "Guard component helps to prevent user from losing unsaved data",
+  "author": "@paprika",
+  "main": "lib/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/guard"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "prop-types": "^15.7.2"
+  },
+  "peerDependencies": {
+    "react": "^16.4.0",
```

Comment:
i think we use 16.8.4

Reply:
We should just peer-depend on 16.8.0 since that's when hooks were introduced